<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="innishanv.corporate.thoughtworks.com" name="scenarios.Chapter_01_TheFirstSeleniumTest" tests="1" failures="1" timestamp="28 Mar 2013 08:59:37 GMT" time="33.025" errors="0">
  <testcase name="testThatResultsAppearForAOneWayJourney" time="33.025" classname="scenarios.Chapter_01_TheFirstSeleniumTest">
    <failure type="org.openqa.selenium.TimeoutException" message="Timed out after 30 seconds waiting for scenarios.Chapter_01_TheFirstSeleniumTest$1@58651e95
Build info: version: &#039;2.31.0&#039;, revision: &#039;1bd294d&#039;, time: &#039;2013-02-27 20:52:59&#039;
System info: os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.8.2&#039;, java.version: &#039;1.6.0_43&#039;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for scenarios.Chapter_01_TheFirstSeleniumTest$1@58651e95
Build info: version: '2.31.0', revision: '1bd294d', time: '2013-02-27 20:52:59'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.8.2', java.version: '1.6.0_43'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:259)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at scenarios.Chapter_01_TheFirstSeleniumTest.waitForSearchResultsToAppear(Chapter_01_TheFirstSeleniumTest.java:93)
	at scenarios.Chapter_01_TheFirstSeleniumTest.testThatResultsAppearForAOneWayJourney(Chapter_01_TheFirstSeleniumTest.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"id","selector":"mod_link"}
Command duration or timeout: 11 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.31.0', revision: '1bd294d', time: '2013-02-27 20:52:59'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.8.2', java.version: '1.6.0_43'
Session ID: 1c0a0071-e21f-1446-9f6a-013627031290
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=MAC, acceptSslCerts=true, javascriptEnabled=true, browserName=firefox, rotatable=false, locationContextEnabled=true, version=19.0.2, cssSelectorsEnabled=true, databaseEnabled=true, handlesAlerts=true, browserConnectionEnabled=true, nativeEvents=false, webStorageEnabled=true, applicationCacheEnabled=true, takesScreenshot=true}]
	at sun.reflect.GeneratedConstructorAccessor15.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:513)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:187)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:145)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:554)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:307)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:348)
	at org.openqa.selenium.By$ById.findElement(By.java:216)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:299)
	at scenarios.Chapter_01_TheFirstSeleniumTest$1.apply(Chapter_01_TheFirstSeleniumTest.java:99)
	at scenarios.Chapter_01_TheFirstSeleniumTest$1.apply(Chapter_01_TheFirstSeleniumTest.java:1)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 26 more
Caused by: org.openqa.selenium.remote.ErrorHandler$UnknownServerException: Unable to locate element: {"method":"id","selector":"mod_link"}
Build info: version: '2.31.0', revision: '1bd294d', time: '2013-02-27 20:52:59'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.8.2', java.version: '1.6.0_43'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///var/folders/3j/6z80gvr17wj5zs7y3hlp6j4m0000gn/T/anonymous2261531308809836941webdriver-profile/extensions/fxdriver@googlecode.com/components/driver_component.js:8281)
	at <anonymous class>.FirefoxDriver.prototype.findElement(file:///var/folders/3j/6z80gvr17wj5zs7y3hlp6j4m0000gn/T/anonymous2261531308809836941webdriver-profile/extensions/fxdriver@googlecode.com/components/driver_component.js:8290)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///var/folders/3j/6z80gvr17wj5zs7y3hlp6j4m0000gn/T/anonymous2261531308809836941webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10272)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///var/folders/3j/6z80gvr17wj5zs7y3hlp6j4m0000gn/T/anonymous2261531308809836941webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10277)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///var/folders/3j/6z80gvr17wj5zs7y3hlp6j4m0000gn/T/anonymous2261531308809836941webdriver-profile/extensions/fxdriver@googlecode.com/components/command_processor.js:10217)
]]>
    </failure>
  </testcase> <!-- testThatResultsAppearForAOneWayJourney -->
</testsuite> <!-- scenarios.Chapter_01_TheFirstSeleniumTest -->
